### ISSUE TITLE
Add League Api

### ISSUE THREAD

**Issue by Talha-T** [jwalton/lol-js#17]
https://tr1.api.riotgames.com/lol/league/v3/leagues/by-summoner/12312323

**Comment by jwalton** [comment]
Yup.  I haven't logged into LoL in a long time though.  :P  If you want to
submit a PR for the new API, I'd certainly be interested in taking it.

**Comment by Talha-T** [comment]
@jwalton I will try, but I cannot probably script coffee files.

**Comment by skylarjhdownes** [comment]
If you want some help getting started with CoffeeScript I'd be happy to assist.  I always meant to come back and contribute to this project.  I probably won't be able to throw a lot of time at it until next month though.
